controller/src/main/java/org/jboss/as/controller/registry/AttributeAccess.java

```diff
@@ -140,9 +140,11 @@ public String toString() {
          */
         ALIAS,
         /**
-         * Force the registration of the attribute regardless of the actual process type.
+         * An attribute which does not require runtime MSC services to be read or written.
+         * This flag can be used in conjunction with STORAGE_RUNTIME to specify that a runtime
+         * attribute can work in the absence of runtime services (e.g. on host controllers).
```

Comment:
Nit: I don't think the "(e.g. on host controllers)" helps much, as an HC has lots of runtime services.